common/utils/hwctrace.h

```diff
@@ -63,8 +68,12 @@ class TraceFunc {
 };
 #define CTRACE() TraceFunc hwctrace(__func__);
 #else
+#ifdef ANDROID
+#define CTRACE() ATRACE_CALL()
```

Comment:
all Android specific stuff to https://github.com/01org/IA-Hardware-Composer/blob/master/os/android/platformdefines.h